### PR TITLE
Small fixes to juju getting started guide

### PR DESCRIPTION
**Pull request by ghodss**
Some small grammatical changes and a command argument fix (reference: https://jujucharms.com/docs/stable/getting-started#create-a-controller).

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1538)

### CODE REVIEW

**Comment by caesarxuchao** [comment]
LGTM. Thanks @ghodss.

File: docs/getting-started-guides/juju.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ and tag the issue with "juju" so we can find them.
 > This documentation focuses on the Juju 2.0 release which will be
 > promoted to stable during the April 2016 release cycle.
 
-To paraphrase, on your local Ubuntu system:  
+To paraphrase, run the following on your local Ubuntu system:
```
Comment (devin-donnelly) [inline docs/getting-started-guides/juju.md]: "To paraphrase" is weird here. I suggest:

"[Install the Juju client](https://jujucharms.com/docs/2.0/getting-started-general) by running the following command on your local Ubuntu system:"

File: docs/getting-started-guides/juju.md [hunk 2]
```diff
@@ -65,17 +65,17 @@ docker run --rm -ti \
 
 ### Configure Juju to your favorite cloud provider
 
-At this point you have access to the Juju client. Before you can deploy a
+At this point, you have access to the Juju client. Before you can deploy a
```
Comment (devin-donnelly) [inline docs/getting-started-guides/juju.md]: "Once you have mounted the container, you should have access to the Juju client."

File: docs/getting-started-guides/juju.md [hunk 3]
```diff
@@ -65,17 +65,17 @@ docker run --rm -ti \
 
 ### Configure Juju to your favorite cloud provider
```
Comment (devin-donnelly) [inline docs/getting-started-guides/juju.md]: ### Configure Juju for your cloud provider

**Comment by devin-donnelly** [comment]
@ghodss , have you made the requested changes? Also, there's now a merge conflict that needs resolving.

**Comment by ghodss** [comment]
This guide was rewritten so these changes are now obsolete.

**Pull request closed without merging**